### ISSUE TITLE
If you try and make a NPC a different size, then reset it using /npc scale 1 nothing happens

### ISSUE THREAD

**Issue by Xemorr** [FancyMcPlugins/FancyNpcs#136]
![image](https://github.com/user-attachments/assets/df0a1d10-fc22-46af-b1e3-57f0320b0f5e)
If you try and make a NPC a different size, then reset it using /npc scale 1 nothing happens

**Comment by Grabsky** [comment]
I'm able to reproduce this on:
```
This server is running Paper version 1.21-78-master@ef96a69 (2024-07-16T08:23:03Z) (Implementing API version 1.21-R0.1-SNAPSHOT)
You are 1 version(s) behind
Download the new version at: https://papermc.io/downloads/paper
Previous version: 1.21-62-5a503d7 (MC: 1.21)
```
You can use `/npc scale (npc) 0.9999999` until this is fixed, or edit `npcs.yml` file manually (be careful with that, as changes might be overriden by the plugin auto-save task.

**Comment by NOIR-DexPrkr** [comment]
My problem is i can't change the scale even in the npcs.yml
Nothing happens, it keeps the same scale.

I tried downgrading the plugin because it worked before. Now i can't even see the command.

**Comment by Xemorr** [comment]
That is an unrelated issue.
________________________________

**Comment by OliverSchlueter** [comment]
Should be fixed in the latest dev build (#208)

**Comment by Xemorr** [comment]
king Oliver 👑
